src/content/nft/index.md

```diff
@@ -0,0 +1,387 @@
+ -->
+
+#### The work in minting an NFT
+
+When you mint an NFT, a few things have to happen:
+
+- it needs to be confirmed as an asset on the blockchain.
+- the owner's account balance must be updated to include that asset. This makes it possible for it to then be traded or verifiably "owned".
+- the transactions that confirm the above need to be added to a block and "immortalised" on the chain.
+- the block needs to be confirmed by everyone in the network as "correct". This consensus is what makes it possible for NFT creators to maximise their earnings – there are no intermediaries needed.
+
+These tasks are done by miners. And they let the rest of the network know about your NFT and who owns it. This means mining needs to be sufficiently difficult, otherwise anyone could just claim that they own the NFT you just minted and fraudulently transfer ownership. There are lots of incentives in place to make sure miners are acting honestly.
+
+[More on mining](/developers/docs/consensus-mechanisms/pow/)
+
+#### Making the NFT tamper-proof
+
+Mining difficulty comes from the fact that it takes a lot of computing power to create new blocks in the chain. Importantly, blocks are created consistently, not just when they're needed. They're created every 12 seconds or so.
+
+This is important for making Ethereum tamper-proof, one of the qualities that makes NFTs possible. The more blocks the more secure the chain. If your NFT was created in block #600 and a hacker were to try and change themselves to the creator of your NFT, it would immediately change the digital fingerprint of all subsequent blocks, #601, #602, #603, and so on. That means anyone running Ethereum software would immediately be able to tell and prevent it from happening.
+
+However this means that computing power needs to be used constantly. It also means that a block that contains 0 NFT transactions will still have roughly the same carbon footprint, because computing power will still be consumed to create it. Other non-NFT transactions will fill the blocks.
+
+#### Blockchains are energy intensive, right now
+
+So yes, there is a carbon footprint associated with creating blocks by mining – and this is a problem for chains like Bitcoin too – but it's not directly the fault of NFTs.
+
+A lot of mining is done using renewable energy sources or untapped energy in remote locations. And there is the argument that the industries that NFTs and cryptocurrencies are disrupting have huge carbon footprints too. But just because existing industries are bad, doesn't mean we shouldn't strive to be better.
+
+And we are. Ethereum is going through some upgrades to make using Ethereum (and by virtue, NFTs) more energy efficient. And that's always been the plan.
+
+### A greener future...
+
+For as long as Ethereum has been around, the energy-consumption of mining has been a huge focus area for developers and researchers. And the vision has always been to replace it as soon as possible.
```

Comment:
The rewards from mining offers financial incentive to seek out and invest in cheaper, more efficient energy solutions, that aren't necessarily tethered to existing dense civilization. Mining rigs can be built and connected to the network nearly anywhere in the world. This investment in renewable energy infrastructure can be developed into the next generation of greener civilizations. 

(another general proof-of-work argument... more applies to Bitcoin long term, but thought this may have a place in this "greener future" section)